src/components/ButtonComponents/NumberButtons/NumberButton.js

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 
-const NumberButton = () => {
+export const NumberButton = (numberKey) => {
```

Comment:
This is going to give us an error. That is because the argument given to our components is *always* the props object. It is going to contain a key called `numberKey` with some value. So that argument, which we should call props, is going to look like `{numberKey: "1"}`. We would access that with `props.numberKey`